src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double firstBase;
+    private double secondBase;
+    private double height;
+
+    public IsoscelesTrapezoid(double firstBase, double secondBase, double height, Color color) {
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.height = height;
+        this.setColor(color);

```

Comment:
better create a constructor in the abstract class, and here call it- `super(color)`